traffic_control/models/common.py

```diff
@@ -89,6 +89,66 @@ class Labels:
         VERY_GOOD = _("Very good")
 
 
+class LaneType(Enum):
+    MAIN = 1
+    FAST = 2
+    TURN_RIGHT = 3
+    TURN_LEFT = 4
+    ADDITIONAL = 5
+    MERGE = 6
+    EXIT = 7
+    WEAVE = 8
+    PUBLIC = 9
+    HEAVY = 10
+    REVERSIBLE = 11
+    LIGHT = 20
+    PEDESTRIAN = 21
+    BIKE = 22
+
+    class Labels:
+        MAIN = _("Main lane")
+        FAST = _("Fast lane")
+        TURN_RIGHT = _("Turn right lane")
+        TURN_LEFT = _("Turn left lane")
+        ADDITIONAL = _("Additional straight lane")
```

Comment:
I'm not sure if `ADDITIONAL` is a good name for this. I'm open for suggestions if there's something more suitable for this.

Reply:
`AUXILIARY` might be a better term for this.